tools/test/topology/test-multi-playback.m4

```diff
@@ -68,19 +68,19 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 
 # playback DAI is SSP TEST_DAI_PORT using 2 periods
 # Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 0
-DAI_ADD(sof/pipe-dai-playback.m4,
-	2, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
-	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
-	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
-#DAI_ADD_SCHED(sof/pipe-dai-sched-playback.m4,
+#DAI_ADD(sof/pipe-dai-playback.m4,
 #	2, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
 #	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
-#	1000, 1, 0, SCHEDULE_TIME_DOMAIN_TIMER,
-#	PIPELINE_PLAYBACK_SCHED_COMP_1)
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+DAI_ADD_SCHED(sof/pipe-dai-sched-playback.m4,
```

Comment:
I'm still not sure for this case we should use DAI_ADD or DAI_ADD_SCHED for the second pipeline?

Reply:
I think this adds the ability to tie the scheduling of the pipelines together. It might be a good way to lookup the id of the additional pipeline and schedule them.